packages/back-end/src/controllers/informationSchemas.ts

```diff
@@ -117,12 +118,20 @@ export async function getTableData(
     return;
   }
 
```

Comment:
Is there a way I could access the datasource's `getFormatDialect` method here instead of manually building the dialect?